README.md

```diff
@@ -31,3 +34,22 @@ HTTPS
 ```
 https://github.com/hmrc/for-frontend.git
 ```
+
```

Comment:
You don't need to include this in the readme as we have a confluence page to cover this: https://confluence.tools.tax.service.gov.uk/display/VOAO/Running+RALD+-+Rent+And+Lease+Details+on+your+local+machine Any additional information can go in this page.